docs/advanced-features/event-driven-packaging.md

```diff
@@ -0,0 +1,262 @@
+<!-- markdownlint-disable -->
+# Event-Driven Packaging (EDP)
+
+> EDP is in private preview. Please contact us for details.
+
+## Overview
+
+Data tend to be created in logical batches by machines, people, and
+pipelines. Detecting these logical events from Amazon S3 events alone is
+complex and requires extensive custom logic.
+
+Quilt's *Event-Driven Packaging* service (EDP) smartly groups one
+or more Amazon S3 object events into a single batch-level event
+in [AWS EventBridge](https://aws.amazon.com/eventbridge/) so that
+you can easily trigger logical events like package creation that
+depend on batches rather than on individual files.
+
+Quilt EDP works with any Amazon service that syncs data to Amazon
+S3 and any services that can be targeted from AWS EventBridge (such
+as AWS Lambda or AWS Batch).
+
+> Any AWS service or action that generates S3 object events may trigger the EDP service.
+
+## Requirements
+
+1. A pre-existing VPC that includes a [NAT
+Gateway](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-nat-gateway.html)
+or the following [VPC
+endpoints](https://docs.aws.amazon.com/vpc/latest/privatelink/concepts.html#concepts-vpc-endpoints):
+  - Amazon S3
+  ([gateway](https://docs.aws.amazon.com/vpc/latest/privatelink/gateway-endpoints.html)
+  or
+  [interface](https://docs.aws.amazon.com/vpc/latest/privatelink/create-interface-endpoint.html))
+  - EventBridge (interface)
```

Comment:
This is missing a link and it's interface endpoint, not "interface"

Reply:
Copied from Sergey's [original docs](https://github.com/quiltdata/edp#readme)

Reply:
Link added